src/olympia/reviewers/views.py

```diff
@@ -1197,6 +1201,10 @@ def clear_pending_rejections(self, request, **kwargs):
         detail=True,
         methods=['get'],
         permission_classes=[AllowAnyKindOfReviewer],
+        authentication_classes=[
```

Comment:
Have you explored what this would look like in the swagger docs? I know we kind of dropped the project of migrating api docs to swagger but could be an interesting exercise to add some anotations to the view class to see if you can explain what you do in the .rst file directly in the swagger.

Reply:
I haven't. It's a bad one for swagger from the start because it doesn't use a serializer and directly returns a `JsonResponse`, so it doesn't automatically document the response (which admittedly only contains a single field). Furthermore it looks like our swagger implementation is not smart enough to know which authentication classes are available for each endpoint, so it was already suggesting to use JWT for that endpoint...

I could add a docstring though.